config/sets/symfony/symfony6/symfony62.php

```diff
@@ -42,78 +19,15 @@
         new AnnotationToAttribute('Sensio\Bundle\FrameworkExtraBundle\Configuration\Template'),
```

Comment:
The Sensio attribute require separate PR for re-take a look as not a symfony component.